download/index.html

```diff
@@ -0,0 +1,130 @@
+        <i class="white apple button icon"></i>
+        MacOS&nbsp;<span class="ui small text">(expect&nbsp;~&nbsp;2021)</span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<h1 class="ui horizontal divider header">Support</h1>
+
+<div class="ui basic very padded segment">
+  <div class="ui doubling center aligned grid">
+    <div class="column">
+      <p>Please report technical issues on <a href="{{ site.github_url }}" target="_blank">Github</a>, and ask for help on <a href="{{ site.reddit_url }}" target="_blank">Reddit</a></p>
+    </div>
+  </div>
+</div>
+
+<h3 class="ui horizontal divider header">Limited Support</h3>
+
+<div class="ui basic very padded segment">
+  <div class="ui placeholder segment" style="display: block;text-align: center;padding: 1em 2.5em;">
+    <span class="ui small text">
+      <span class="ui red text" style="font-size: 1.15em;"><b>Warning:</b></span> These platforms are compatible, but not all functionality is implemented or tested. <b>Probability of encountering unknown issues is high.</b> UI components may be impaired or completely broken. Issues we know about are listed:
+    </span>
+  </div>
+
+  <div class="ui basic segment">
+    <div class="row">
+      <div class="ui two column very relaxed grid">
+        <div class="column">
+          <div class="ui center aligned grid">
+            <div class="row">
+              <div class="sixteen wide column">
+                <h4>Earlier Windows</h4>
+                <span class="ui small text">
+                  <i class="info circle icon"></i> Core functional and supported.<br>
+                  Notifications and Prompts are not implemented. For the best experience, do not set the default action to "ask" until a workaround is available.
+                </span>
+              </div>
+            </div>
+            <div class="two column doubling row">
+              <div class="column">
+                <a href="https://updates.safing.io/latest/windows_amd64/packages/portmaster-installer.exe" target="_blank" class="ui button">
+                  <i class="windows icon"></i>
+                  Windows&nbsp;7
+                </a>
+              </div>
+              <div class="column">
+                <a href="https://updates.safing.io/latest/windows_amd64/packages/portmaster-installer.exe" target="_blank" class="ui button">
+                  <i class="windows icon"></i>
+                  Windows&nbsp;8
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="column">
+          <div class="ui center aligned grid">
+            <div class="row">
+              <div class="sixteen wide column">
+                <h4>Linux 2.4+</h4>
+                <span class="ui small text">
+                  <i class="info circle icon"></i> Core functional and supported.<br>
+                  Desktop environment integration varies heavily. Currently only packaged for Debian and its derivatives.
```

Comment:
Please also add note:
`You can contribute to your favourite distro's installer <a href="https://github.com/safing/portmaster-packaging">here</a>.`
or something like that.

Reply:
with the re-defined download page and link to the wiki, should this note be in the wiki or on the download page

Reply:
👆 @dhaavi 😉